Resources/Locale/en-US/round-end/round-end-system.ftl

```diff
@@ -7,5 +7,6 @@ round-end-system-shuttle-recalled-announcement = The emergency shuttle has been
 round-end-system-shuttle-sender-announcement = Station
 round-end-system-round-restart-eta-announcement = Restarting the round in {$time} {$units}...
 
+eta-units-minute = minute
 eta-units-minutes = minutes
 eta-units-seconds = seconds
```

Comment:
```suggestion
round-end-system-round-restart-eta-announcement = Restarting the round in {$time} {$units}...

eta-units-minutes = {$amount ->
    [one] minute
    *[other] minutes
}
eta-units-seconds = {$amount ->
    [one] second
    *[other] seconds
}
```
Note that this will require the units's `Loc.GetString(unitsLocString)` to supply time as well. Ideally you would change it all to be a bit better somehow, e.g. have the time string generated separately and just substituted into the "restarting the round..." 

Also this sorta "minutes" and "seconds" stuff probably should be moved into being its own thing somewhere and have this just reference it somehow. But dunno if this number stuff happens often enough to warrant it.